app/views/businesses/show.html.erb

```diff
@@ -1,13 +1,18 @@
-<%= link_to "Update Business Information", edit_business_path(@business.slug) %>
 <div class="business_info">
   <h1><%= @business.name %></h1>
   <%= @business.description %>
 </div>
 
+<% if current_user %>
+  <% if current_user.admin? || current_user.platform_admin? %>
```

Comment:
I think we can consolidate this code. I think we have an admin method that checks for business admin and platform admin at the same time.